### DISCUSSION TITLE
No ply download option

### DISCUSSION THREAD

**Discussion by ruijenyang** [thygate/stable-diffusion-webui-depthmap-script#210] [Q&A] [answered]
I try to use the in-paint model in Blender, but it failed with the obj file. So I try to add the option in the setting to generate a ply file.
But there is still no ply file to be downloaded, only an obj file...... 
![Screenshot 2023-05-07 141041](https://user-images.githubusercontent.com/119155130/236695097-12230e93-84f0-441a-889f-7e1ee76286ee.png)
![Screenshot 2023-05-07 141236](https://user-images.githubusercontent.com/119155130/236695180-0d6824fa-1335-4db0-a860-9941960e5099.png)

**Comment by thygate** [comment, marked as answer]
The files are saved in the /outputs/extras-images directory

**Reply by thygate** [reply]
The OBJ file should import fine in blender, just don't use the legacy wavefront importer.

**Reply by ruijenyang** [reply]
Solve it!!!!!! Save my final. Thanks a lot!

**Reply by f-rank** [reply]
> Solve it!!!!!! Save my final. Thanks a lot!

@ruijenyang I imported it, not with the legacy one, and it looks like points on the color info. Did you have to do anything other than the specific importer for it to look ok ?

Am getting stuff like this.
![image](https://user-images.githubusercontent.com/1836566/236701305-8b3c4eb1-4715-4ea3-ad9f-12435f4d785a.png)

**Reply by ruijenyang** [reply]
Use the ply format to import. Obj files have this kind of issue. 
Also, check discussion #181
https://github.com/thygate/stable-diffusion-webui-depthmap-script/discussions/181#discussioncomment-5610697
